IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewControllers/BaseViewController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  BaseViewController.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/21/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController, BaseViewControllerProtocol {
```

Comment:
This doesn't seem to be used. (but I remember you used it before.. is this class still required?)